src/eth_provider/provider.rs

```diff
@@ -458,10 +456,14 @@ where
             })
             .collect::<Vec<_>>();
 
+        let mut base_fee_per_gas =
+            blocks.iter().map(|header| header.header.base_fee_per_gas.unwrap_or_default()).collect::<Vec<_>>();
+        base_fee_per_gas.extend_from_within((base_fee_per_gas.len() - 1)..);
```

Comment:
so we take the last element of the vec and append it to the vec? Is that the spec?

Reply:
no no the spec is to check if gas ratio > 50% then bump base fee per gas by 12.5% probably

Reply:
ah ok ok but for now you just append the last one?